Improving documentation (#85)
StenopePHP/Stenope: doc/processors.md

# Content processors

Content processor are the main entry point for custom behaviours in Stenope.

## What are processors

> Processors intervene in the content loading process, after the raw content is parsed and normalized into an array, and before it's hydrated into a model object.

Reviewer: To shorten/rephrase:

```suggestion
A processor acts on the raw content loaded from your sources as an array, before denormalization into its model.
```

?

> Writing a custom processor allow you to aply virtualiy any modification to the normalized content before it's returned as an object.

Reviewer: ```suggestion
Writing a custom processor allow you to apply virtually any modification to the normalized content before denormalization.
```

> Internaly, Stenope has processors for:

Reviewer: ```suggestion
Internally, Stenope registers default processors for:
```

> - Highlighting code blocks with synthaxic coloration.

Reviewer: ```suggestion
- Syntax highlighting of code blocks.
```
